app/models/user.js

```diff
@@ -0,0 +1,34 @@
+import DS from 'ember-data';
+import { attr, hasMany } from '@ember-decorators/data';
+
+const { Model } = DS;
+
+export default class UserModel extends Model {
+// DB table: dcp_projectlupteam
+
+  // user's internal id should be their email
+
+  // One Project Has Many Users
+  @hasMany('project') projects;
+
```

Comment:
Users can also have many projects. Is that adequately captured as is?

Reply:
Sorry the comment seems to be out of order but the relationship is coded correctly. Will alter

Reply:
Should be captured. Actually, we may even get rid of the User model's `@hasMany('project') projects;` and the Project model's `@hasMany('user') users;`, since those associated can be acquired through `userProjectParticipation`